src/Facility.Definition/Fsd/FsdParsers.cs

```diff
@@ -40,7 +40,7 @@ internal static class FsdParsers
 			parser.Bracketed(PunctuationParser(openBracket), PunctuationParser(closeBracket));
 
 		private static IParser<Match> AttributeParameterValueParser { get; } =
-			Parser.Regex(@"""(([^""\\]+|\\[""\\/bfnrt]|\\u[0-9a-fA-f]{4})*)""|([0-9a-zA-Z.+_-]+)");
+			Parser.Regex(@"""(([^""]+|\\[""\\/bfnrt]|\\u[0-9a-fA-f]{4})*)""|([0-9a-zA-Z.+_-]+)");
```

Comment:
```suggestion
		private static IParser<Match> AttributeParameterValueParser { get; } =
			Parser.Regex(@"""(([^""]+)*)""|([0-9a-zA-Z.+_-]+)");
```

Or even simpler, if we can get rid of that test. It's the only one that fails with this regex, too.